bookinfo-example/authservice/templates/deployment.yaml

```diff
@@ -29,6 +29,9 @@ spec:
         volumeMounts:
           - name: authservice-config # mount the volume containing the authservice ConfigMap
             mountPath: /etc/authservice
+        livenessProbe:
```

Comment:
I think we should use readinessProbe. Liveness probe will continuously be checked by k8s and if fails, pod is killed. however, we only care about the pod startup time, authsvc is able to fetch the jwks public key.